index.ts

```diff
@@ -348,24 +348,24 @@ async function init() {
   if (needsPinia) {
     render('config/pinia')
   }
-  if (needsVitest) {
-    render('config/vitest')
-  }
   if (needsCypress) {
     render('config/cypress')
   }
-  if (needsCypressCT) {
-    render('config/cypress-ct')
-  }
   if (needsNightwatch) {
     render('config/nightwatch')
   }
-  if (needsNightwatchCT) {
-    render('config/nightwatch-ct')
-  }
   if (needsPlaywright) {
     render('config/playwright')
   }
+  if (needsVitest) {
+    render('config/vitest')
+  }
+  if (needsCypressCT) {
+    render('config/cypress-ct')
+  }
+  if (needsNightwatchCT) {
+    render('config/nightwatch-ct')
+  }
```

Comment:
I don't mind switching the order but I'm afraid this might break something. But if the `pnpm test` script still work then it should be OK.